apps/coscrad-frontend/src/components/resources/spatial-features/utils/identify-centre-point-for-map.ts

```diff
@@ -0,0 +1,10 @@
+import { findFirstPoint } from './find-first-point';
+
+/**
+ * This utility function is used to centre the map on a specific spatial feature.
+ * We currently just centre the map on the first point in the feature, but
+ * we can update this logic to use the feature's geometric centre in the future.
+ */
+export const identifyCentrePointForMap = (
```

Comment:
Ideally, we shouldn't need this. Can't the map do this for us? Can we at least hide this behind the map implementation?

Reply:
comment added to this effect